src/components/App.tsx

```diff
@@ -131,31 +135,46 @@ class App extends Component<Props> {
   render() {
     return (
       <MainContainer>
-        {this.props.view === 'browser' && (
-          <Browser
-            onRunQuery={() => this.props.runQuery({ query: this.props.query })}
-            onSelectQuery={(queryName) => {
-              this.props.resetQueryResults();
-              this.props.selectSavedQuery(queryName, this.props.autorunQuery);
-            }}
-            onEditQuery={(queryName) => {
-              this.props.editQuery(queryName);
-            }}
+        <Switch>
+          <Route
+            exact
+            path={ROUTES.BROWSER}
+            render={() => (
```

Comment:
Just a suggestion - but in my opinion using `children` instead of `render` could make this code a bit cleaner by using composition.

Reply:
Good idea